### ISSUE TITLE
Dropdown: Accessibility error - Form Elements must have labels

### ISSUE THREAD

**Issue by donSan4o** [primefaces/primereact#7199]
### Describe the bug

Message from axe DevTools for Chrome:
" Ensure every form element has a label

more informationLink opens in a new window
Element Location: input[value="Rome"]

<input type="text" readonly="" aria-haspopup="listbox" tabindex="0" data-pc-section="input" value="Rome">

To solve this problem, you need to fix at least (1) of the following:
Form element does not have an implicit (wrapped) <label>
Form element does not have an explicit <label>
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element has no placeholder attribute
Element's default semantics were not overridden with role="none" or role="presentation"
"

### System Information

```Shell
primereact 10.8.3
```

### Expected behavior

need to add 'aria-label' attr
